include/kernels/staggered_coarse_op_kernel.cuh

```diff
@@ -0,0 +1,316 @@
+#include <color_spinor_field_order.h>
```

Comment:
Remind me, why keep these kernels in a separate file vs `coarse_op_kernel.cuh`?

Reply:
Mostly historical at this point. The main blocker for a while has been that `coarse_op_kernel.cuh` is long/confusing enough as it is, and because the routines in this driver compile quickly enough the benefit of `jitify` support wasn't a good incentive. If you'd really rather I merged it into `coarse_op_kernel.cuh`, I will.

Reply:
No need.  Just asking.